android/app/src/main/AndroidManifest.xml

```diff
@@ -3,12 +3,7 @@
 
     <!-- Local Notifications -->
     <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED"/>
```

Comment:
just a note that there are other permissions mentioned in the readme that hadn't been added here